### PR TITLE
Ground work for shared example app

### PR DESCRIPTION
**Pull request by gumil**
- Add Xamarin Forms sample app
 - Implement PCL Bait and Switch trick for iOS and Android bridge

### CODE REVIEW

File: UsabillaBindings/Xamarin.Usabilla.Sample.Android/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Xamarin.Usabilla.Sample.Android")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Xamarin.Usabilla.Sample.Android")]
+[assembly: AssemblyCopyright("Copyright ©  2014")]
```
Comment (andlie) [inline UsabillaBindings/Xamarin.Usabilla.Sample.Android/Properties/AssemblyInfo.cs]: maybe 2019 is the right year?

File: UsabillaBindings/Xamarin.Usabilla.Sample.iOS/Info.plist [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    </array>
+    <key>UISupportedInterfaceOrientations~ipad</key>
+    <array>
+        <string>UIInterfaceOrientationPortrait</string>
+        <string>UIInterfaceOrientationPortraitUpsideDown</string>
+        <string>UIInterfaceOrientationLandscapeLeft</string>
+        <string>UIInterfaceOrientationLandscapeRight</string>
+    </array>
+    <key>MinimumOSVersion</key>
+    <string>8.0</string>
```
Comment (andlie) [inline UsabillaBindings/Xamarin.Usabilla.Sample.iOS/Info.plist]: Our SDK only supports 9.0+, so we should say MinimumOSVersion 9.0

File: UsabillaBindings/Xamarin.Usabilla.Sample.iOS/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Xamarin.Usabilla.Sample.iOS")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Xamarin.Usabilla.Sample.iOS")]
+[assembly: AssemblyCopyright("Copyright ©  2014")]
```
Comment (andlie) [inline UsabillaBindings/Xamarin.Usabilla.Sample.iOS/Properties/AssemblyInfo.cs]: again 2019....

**Comment by kioli** [comment]
Has the folder structure been checked and validated with Anders?
I mean the presence of the folders `Framework_WithUpdates` and `images` in the top layer

File: UsabillaBindings/Xamarin.Usabilla.Sample.Android/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (gumil) [reply UsabillaBindings/Xamarin.Usabilla.Sample.Android/Properties/AssemblyInfo.cs]: Auto generated but nice catch

**Comment by gumil** [comment]
> Should we change the bin/**debug**. to bin/**release** to make sure we build the Nuget Package off the release build of our dll's

I agree with this. But the current setup there's no difference between the debug and release binary as we don't do signing, at least for Android.

**Pull request merged**